guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
 
-### Compatible with Kubernetes
+All AI models (PyTorch, TensorFlow, TensorRT, etc.) SHOULD be in separate sub-folders of the `/opt/holoscan/models/` folder. In specific use cases where the app package developer is prevented from enclosing the model files in the package/container due to intellectual property concerns, the models can be supplied from the host system when the app package is run, e.g., via the volume mount mappings and the use of container env variables.
 
-A MAP SHALL NOT be in a format which inhibits or hampers the ability to deploy it using Kubernetes.
+## Manifests
 
+A MAP SHALL contain two manifests: the application manifest and the package manifest. The package manifest shall be stored in `/etc/holoscan/pkg.json`, and the application manifest shall be stored in `/etc/holoscan/app.json`. Once a MAP is created, its manifests are immutable.
 
-### OCI Compliance
+### Application Manifest
 
-The containerized portion of a MAP SHALL comply with [Open Container Initiative](https://opencontainers.org/) format standards.
+The Application Manifest file provides information about the MAP's Application.
 
+- Application Manifest MUST define the type of the containerized application (`/etc/holoscan/app.json#type`).
 
-### Facilitate GPU Acceleration
+  - Type SHALL have the value of either `service` or `application.`
 
-A MAP SHALL enable applications to be developed with GPU acceleration.
+- Application Manifest MUST define the command used to run the Application (`/etc/holoscan/app.json#command`).
 
-A MAP SHALL NOT impede the adoption or utilization of accelerator (GPU) technology.
+  - When not provided, the MAP will be considered invalid and not runnable by MONAI Deploy Application Runner and hosting services that comply to MAP specification.
 
+- Application Manifest SHOULD define the version of the manifest file schema (`/etc/holoscan/app.json#api-version`).
 
-# Architecture & Design
+  - Manifest schema version SHALL be provided as a [semantic version](https://semver.org/) string.
 
-##  Description
+  - When not provided, the default value `0.0.0` SHALL be assumed.
 
-The MONAI Application Package (MAP) is a functional package designed to perform an action on datasets of a prescribed format. A MAP is a container image which adheres the specification provided in this document.
+- Application Manifest SHOULD define the version of the MONAI Deploy SDK used to create the Application (`/etc/holoscan/app.json#sdk-version`).
 
+  - SDK version SHALL be provided as a [semantic version](https://semver.org/) string.
 
-## Application
+  - When not provided, the default value `0.0.0` SHALL be assumed.
 
-The primary component of a MAP is the Application. The Application is provided by an application developer and incorporated into the MAP using the MONAI Package Builder (Builder).
+- Application Manifest SHOULD define the version of the application itself (`/etc/holoscan/app.json#version`).
```

Comment:
How is this related or different from the earlier red `org.opencontainers.image.version: A MAP container image SHALL provide a version of the packaged application using the semantic versioning format.`

Should it be clarified that the org.opencontainers.image.version is for container version, different from the version mentioned here?
If the same, please also state the duplication.

Reply:
Updating to state that `org.opencontainers.image.version` is the same as the application.